pkg/service/roomservice_test.go

```diff
@@ -42,11 +44,62 @@ func TestDeleteRoom(t *testing.T) {
 	})
 }
 
+func TestMetaDataLimits(t *testing.T) {
+	t.Run("metadata exceed limits", func(t *testing.T) {
+		svc := newTestRoomService()
```

Comment:
Maybe worth passing in the max metadata size here and check with 0 value also to make sure that no limit works when it is set to 0.

Reply:
Done